### ISSUE TITLE
Remove redundant type compiler overrides

### ISSUE THREAD

**Issue by kadler** [IBM/sqlalchemy-ibmi#129]
The DB2TypeCompiler class needlessly overrides various types with the same definition from the GenericTypeCompiler parent class.

eg.
```python
    def visit_SMALLINT(self, type_, **kw):
        return "SMALLINT"

    def visit_INT(self, type_):
        return "INT"

    def visit_BIGINT(self, type_, **kw):
        return "BIGINT"
```

These are all exactly the same as in the [parent class](https://github.com/sqlalchemy/sqlalchemy/blob/fb81f9c8d914f9911925dd3f4e77d7fc374b267c/lib/sqlalchemy/sql/compiler.py#L4502-L4509) so why are we overriding them? It just adds noise to the code.

**Comment by qtemp** [comment]
There is no parent class visit_INT; I only see a visit_INTEGER?

**Comment by kadler** [comment]
Fixed in 92e1df5affa1327b60d777d73bc6d97c3cf6f0fd
